src/Storage/Device.php

```diff
@@ -51,21 +52,26 @@ abstract public function getPath(string $filename): string;
      *
      * @param string $source
      * @param string $path
-     *
+     * @param int $chunk
+     * @param int $chunks
+     * @param string $tmp
+     * 
      * @throws \Exception
      *
-     * @return bool
+     * @return int
```

Comment:
The returned `bool` didn't need much explanation, but I think an `int` return value needs more detail.

Reply:
updated the description